spikeinterface/toolkit/preprocessing/resample.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import scipy.signal
+
+from .basepreprocessor import (
+    BasePreprocessor,
+    BasePreprocessorSegment,
+)
+from .filter import fix_dtype
+
+try:
+    from scipy import special, signal
+
+    HAVE_RR = True
+except ImportError:
+    HAVE_RR = False
```

Comment:
you can safely remove the try except since the `toolkit` module depends already on `scipy`

Reply:
Did them on the new pull.